api/src/main/java/uk/gov/hmcts/reform/divorce/caseapi/config/EmailTemplatesConfig.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.divorce.caseapi.config;
+
+import lombok.Getter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.annotation.Validated;
+import uk.gov.hmcts.reform.divorce.caseapi.enums.LanguagePreference;
+
+import java.util.HashMap;
+import java.util.Map;
+import javax.validation.constraints.NotNull;
+
+@Component
+@ConfigurationProperties(prefix = "uk.gov.notify.email")
+@Validated
+@Getter
+@SuppressWarnings("PMD") // UseConcurrentHashMap
+public class EmailTemplatesConfig {
+    @NotNull
+    private Map<LanguagePreference, Map<String, String>> templates = new HashMap<>();
```

Comment:
I really don't like to be that guy, but can we use final here? If possible I'd like to default to it